local/gugcat/index.php

```diff
@@ -147,8 +149,7 @@
 $columns = grade_capture::get_columns();
 
 echo $OUTPUT->header();
-if(!empty($activities))
```

Comment:
Why is it the checker was removed again? Set_cm will throw an error if there is no selectedmodule.